python/libcudf/libcudf/load.py

```diff
@@ -16,8 +16,36 @@
 import ctypes
 import os
 
+# Loading with RTLD_LOCAL adds the library itself to the loader's
+# loaded library cache without loading any symbols into the global
+# namespace. This allows libraries that express a dependency on
+# this library to be loaded later and successfully satisfy this dependency
+# without polluting the global symbol table with symbols from
+# libcuspatial that could conflict with symbols from other DSOs.
```

Comment:
```suggestion
# libkvikio that could conflict with symbols from other DSOs.
```

(will commit once CI is done running on https://github.com/rapidsai/cuspatial/pull/1483)